packages/ra-input-rich-text/src/index.tsx

```diff
@@ -1,34 +1,68 @@
 import debounce from 'lodash/debounce';
-import React, { useRef, useEffect, useCallback } from 'react';
-import PropTypes from 'prop-types';
-import Quill from 'quill';
+import React, {
+    useRef,
+    useEffect,
+    useCallback,
+    FunctionComponent,
+    ComponentProps,
+} from 'react';
+import Quill, { QuillOptionsStatic } from 'quill';
 import { useInput, FieldTitle } from 'ra-core';
 import { InputHelperText } from 'ra-ui-materialui';
-import { FormHelperText, FormControl, InputLabel } from '@material-ui/core';
+import {
+    FormHelperText,
+    FormControl,
+    InputLabel,
+    PropTypes,
+} from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
 import styles from './styles';
 
 const useStyles = makeStyles(styles, { name: 'RaRichTextInput' });
 
-const RichTextInput = ({
-    options = {}, // Quill editor options
-    record = {},
-    toolbar = true,
-    fullWidth = true,
-    configureQuill,
-    helperText = false,
-    label,
-    source,
-    resource,
-    variant,
-    margin = 'dense',
-    ...rest
-}) => {
-    const classes = useStyles();
-    const quillInstance = useRef();
-    const divRef = useRef();
-    const editor = useRef();
+interface IProps {
```

Comment:
We iue the name "Props" in the react-admin codebase for the props interface. Can you remove the leading 'I' in your interface?